### ISSUE TITLE
Custom Loadout Stows Improperly

### ISSUE THREAD

**Issue by Davnik** [HeavensGate/Eternal#311]
When your custom loadout spawns in, it doesn't default to your backpack. Some things go in [the item which spawns in your hand.] This literally only comes up for medical staff, and maybe IAA's, so it's low priority, but it is a known fallacy. When your custom loadout spawns in, it doesn't default to your backpack. Some things go in [the item which spawns in your hand.] This literally only comes up for medical staff, and maybe IAA's, so it's low priority, but it is a known fallacy.

**Comment by aztectornado** [comment]
So do antag items, such as cultist tomes/notes. I think the code just grabs the first container it can find.

**Comment by Puckaboo2** [comment]
Yeah, I noticed this when I spawn as a surgeon; my purple cap spawns in my emergency medical kit, of all things.
